src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "skipRestore": {
+      "longName": "no-restore",
+      "shortName": ""
+    },
+    "Hosted": {
+      "longName": "hosted"
+    },
+    "PWA": {
+      "longName": "pwa"
+    },
+    "Framework": {
+      "longName": "framework"
+    },
```

Comment:
What does this arg do, do we still need it?

Reply:
It allows the user to choose the target framework for the project. I don't think it is necessary, we can remove it for the empty template if we are okay with the default framework being .NET 7.0.

Reply:
So is it essentially a symbol for just one option (.NET 7) right now? Maybe we should keep it in place for future .NET versions to simplify things in the future?

Reply:
Yes, just one option. It would probably be best to keep the framework argument, especially if we plan to add more options in the future.